docs/getting-started-on-snowplow-bdp-cloud/first-party-tracking/index.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "First party tracking"
+sidebar_position: 3
+sidebar_custom_props:
+  offerings:
+    - cloud
+---
+
+# First party tracking
+
+This feature lets you configure a custom collector domain to match the domain you wish to track. This allows for first-party cookies to be set on the server.
+Enter the domain you wish to track, **any sub domain or multi level sub domain will be removed** and a cookie domain will be generated. If you wish to restrict the tracking to a sub domain the cookie domain can still be edited.
+
+![enter_domain](images/Screenshot_enter_domain.png)
+
+## DNS records
+The required DNS records will now be created, this may take several minutes.
+When the records are ready you will receive email confirmation.
+
+![create_dns_records](images/Screenshot_create_dns_records.png)
+
+Once the DNS records are available, copy them into your domain provider.
+We will attempt to verify that the records have been correctly applied up to 72 hours after they have been generated.
```

Comment:
What does this mean? The first time I read it I interpreted it as “at a single arbitrary point between 0 and 72 hours after generating the records, we will attempt to verify them”. But now I think you probably want to say that we will be continuously trying to verify within the next 72 hours?